manifests/init.pp

```diff
@@ -312,9 +312,10 @@
 #   one processor on receipt of the token. The max_messages parameter is limited
 #   to 256000 / netmtu to prevent overflow of the kernel transmit buffers.
 #
-# @param test_corosync_config
```

Comment:
`test_corosync_config` is still a parameter though, so doesn't it also need documenting?

Reply:
Did you mean to remove the documentation for `test_corosync_config_cmd` instead?

(Also, do you want to check the email address you use for your commits so that it links to your github user?)

Reply:
excuse me, you are right